### PR TITLE
Added article for multitargeting using your project file

### PR DESCRIPTION
**Pull request by Mikejo5000**
The existing topic on multitargeting, https://docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks, appears to be specific to non-SDK-style. We've talked about the need for an article with updated content.

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[16bbd09](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1540/commits/16bbd0922d693c2e53cf516a402b5f357b5c3fb6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1540) |
[docs/create-packages/multitargeting-sdk-projects.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/multitargeting-sdk-projects.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multitargeting-sdk-projects?branch=pr-en-us-1540) |
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1540) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907161924132192-1540%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=wwXik6aCTfnQ5DJSGfHHBlDT4L8ThlkzwkI3F8%2Fj7dQ%3D&st=2019-07-16T19%3A20%3A12Z&se=2019-08-16T19%3A25%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[944c943](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1540/commits/944c94356a1deac4aa30da7be41d564adba9f2ba)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/TOC.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) | [Details](#user-content-571bd1ef-9bb1063d)
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1540) | [Details](#user-content-9078d347-9bb1063d)
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1540) | [Details](#user-content-d9a31f79-9bb1063d)
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |

<a id="571bd1ef-9bb1063d"></a>
### [docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/TOC.md)
 - **Line 0**: **[Warning]** `````Invalid file link:(~/multiple-target-frameworks-project-file.md).`````

<a id="9078d347-9bb1063d"></a>
### [docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md)
 - **Line 12**: **[Warning]** `````Invalid file link:(~/create-packages/multitargeting-sdk-projects.md).`````

<a id="d9a31f79-9bb1063d"></a>
### [docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md)
 - **Line 15**: **[Warning]** `````Invalid file link:(~/create-packages/multitargeting-sdk-projects.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907161926534454-1540%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=aLNThQACMQ14YNCNQ2SXAWQRg%2BdMddaxPNmiE2RHSBk%3D&st=2019-07-16T19%3A22%3A48Z&se=2019-08-16T19%3A27%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[dd6f028](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1540/commits/dd6f0286b54a0a0bc83b56788ff1223ce9f7292d)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1540) | [Details](#user-content-9078d347-37e12d2f)
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1540) |

<a id="9078d347-37e12d2f"></a>
### [docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md)
 - **Line 12**: **[Warning]** `````Invalid file link:(~/create-packages/support-multiple-target-frameworks-project-file.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907161945407538-1540%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AGk8lm7FUBP13OcygFw%2FtBB604b0CLRX5Be%2BEpuUmJI%3D&st=2019-07-16T19%3A41%3A37Z&se=2019-08-16T19%3A46%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[cc0c9ae](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1540/commits/cc0c9aefe31e763cca82bfa4aba474e5e8de25a7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1540) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1540) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907161950252259-1540%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2Bxj3mkwUCuT2px3WRsxlOpP%2B4%2BGE5jvXw%2Btf6VqaOqI%3D&st=2019-07-16T19%3A46%3A22Z&se=2019-08-16T19%3A51%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Mikejo5000** [comment]
Most of the article is sourced from Andy's comments in #859

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[c75aeeb](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1540/commits/c75aeebda4f13ac63fccd4e65c53b1191008c6d8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1540) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1540) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907162300136553-1540%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2BRLtPHhCn691%2BdAi3lqa1EAYIh9L1hMMqObDMAjzgis%3D&st=2019-07-16T22%3A56%3A17Z&se=2019-08-16T23%3A01%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/multiple-target-frameworks-project-file.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+   #else
+   #error This code block does not match csproj TargetFrameworks list
+   #endif
+      }
+   }
+   ```
+
+4. Add any NuGet metadata you want to the *.csproj* as MSBuild properties.
+
+   For the list of available metadata and the MSBuild property names, see [pack target](../reference/msbuild-targets.md#pack-target). Also see [Controlling dependency assets](../consume-packages/package-references-in-project-files.md#controlling-dependency-assets).
```
Comment (zivkan) [inline docs/create-packages/multiple-target-frameworks-project-file.md]:
```suggestion
   For the list of available package metadata and the MSBuild property names, see [pack target](../reference/msbuild-targets.md#pack-target). Also see [Controlling dependency assets](../consume-packages/package-references-in-project-files.md#controlling-dependency-assets).
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[67f4ede](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1540/commits/67f4edece6f961d8785f2f588437067dd1913aed)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1540) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1540) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907172130194696-1540%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FckigBlrwSbxlNsN1YIQymH195gh8AhVlt1tSxeqoE0%3D&st=2019-07-17T21%3A26%3A28Z&se=2019-08-17T21%3A31%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[6447075](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1540/commits/644707513e802da351ea1d7db64230586035fe4a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1540) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1540) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907172130339981-1540%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2yTrQ4ELdNtJ04N7WE1sgytpApO50aScm5cD5iEFSuY%3D&st=2019-07-17T21%3A27%3A32Z&se=2019-08-17T21%3A32%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Mikejo5000** [comment]
@zivkan Sounds like the existing page also applies to SDK-style projects, if one chooses to do the manual setup. Is that right?

**Comment by zivkan** [comment]
@Mikejo5000 yes, the existing page describes how the package needs to be laid out, so it applies regardless of how the package or assemblies are built (multiple non-SDK csproj files and custom nuspec file, or a single multi-targeting SDK style csproj). The NuGet pack targets coupled with SDK style projects know about this layout and automate putting the assemblies in the correct lib folders and creating dependency groups for each TFM (info about this appears to be missing, I'll create an issue so we don't forget)

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[451c466](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1540/commits/451c466890418f13be118b60951125b75f42423d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1540) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1540) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907172239185863-1540%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=rMIuc2cuIAth3QeJaODmPmTj60gV9mYDuLQS72RC5Yk%3D&st=2019-07-17T22%3A35%3A12Z&se=2019-08-17T22%3A40%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/multiple-target-frameworks-project-file.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+   We recommend that you create a .NET Standard class library for best compatibility.
+
+2. Edit the *.csproj* file to support the target frameworks.
+
+   For example, change `<TargetFramework>netstandard2.0</TargetFramework>` to `<TargetFrameworks>netstandard2.0;net45</TargetFrameworks>`.
+
+   Make sure that you change the XML element changed from singular to plural (add the "s" to both the open and close tags).
+
+3. If you have any code that only works in one TFM, you can use `#if NET45` or `#if NETSTANDARD20` to separate TFM-dependent code. For example, you can use the following code:
```
Comment (nkolev92) [inline docs/create-packages/multiple-target-frameworks-project-file.md]: Link https://docs.microsoft.com/en-us/dotnet/core/tutorials/libraries#how-to-multitarget maybe?

File: docs/create-packages/multiple-target-frameworks-project-file.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+   #endif
+      }
+   }
+   ```
+
+4. Add any NuGet metadata you want to the *.csproj* as MSBuild properties.
+
+   For the list of available package metadata and the MSBuild property names, see [pack target](../reference/msbuild-targets.md#pack-target). Also see [Controlling dependency assets](../consume-packages/package-references-in-project-files.md#controlling-dependency-assets).
+
+   If you want to separate build-related properties from NuGet metadata, you can use a different `PropertyGroup`, or put the NuGet properties in another file and use MSBuild's `Import` directive to include it.
```
Comment (nkolev92) [inline docs/create-packages/multiple-target-frameworks-project-file.md]: Directory.Build.Props and Directory.Build.Targets are also supported with 15.x. 
Worth calling out.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[be26f02](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1540/commits/be26f029c60decf433680ebe686009cd9a6abaee)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/create-packages/Supporting-Multiple-Target-Frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/Supporting-Multiple-Target-Frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks?branch=pr-en-us-1540) |
[docs/create-packages/multiple-target-frameworks-project-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/create-packages/multiple-target-frameworks-project-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/multiple-target-frameworks-project-file?branch=pr-en-us-1540) |
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-multi/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1540) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C18%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907182304108723-1540%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=HOtNeqduFBFo9CLN9wngcNlwmbKr8a7J4B8ltavLXmk%3D&st=2019-07-18T23%3A00%3A05Z&se=2019-08-18T23%3A05%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**